Procfile

```diff
@@ -1,2 +1,2 @@
 web: bin/rails server -p $PORT -e $RAILS_ENV
-release: bin/rails db:migrate && bin/rails db:seed
+release: bin/rails db:migrate && bin/rails db:seed && bin/rails r -e production lib/production_tasks.rb
```

Comment:
Zastanawiałem się jakie rozwiązanie bardziej stosuje się produkcyjnie. Czy lepiej to np. zrobić z console w produkcji, czy właśnie przez runnera. Za console przemawia to że tak naprawdę wystarczy jedno wywołanie po wdrożeniu feature-a, dla starszych rekordów, a runner będzie niestety leciał za każdym razem przy deployu. Mimo to wydaje mi się że jest to pewniejsze rozwiązanie.